vms/platformvm/client_permissionless_validator.go

```diff
@@ -44,9 +44,13 @@ type ClientPermissionlessValidator struct {
 	PotentialReward        *uint64
 	AccruedDelegateeReward *uint64
 	DelegationFee          float32
-	Uptime                 *float32
-	Connected              *bool
-	Signer                 *signer.ProofOfPossession
+	// Deprecated: Uptime is deprecated for Subnet Validators.
+	//						 It will be available only for Primary Network Validators.
+	Uptime *float32
+	// Deprecated: Connected is deprecated for Subnet Validators.
+	//						 It will be available only for Primary Network Validators.
```

Comment:
These comments look pretty weird. Was this intended?

Reply:
nope, seems it used tabs instead of spaces